app/dts/layouts/common/60percent/60percent.dtsi

```diff
@@ -0,0 +1,72 @@
+#include <layouts/common/60percent/ansi.dtsi>
+#include <layouts/common/60percent/iso.dtsi>
+#include <layouts/common/60percent/all1u.dtsi>
+#include <layouts/common/60percent/hhkb.dtsi>
+
+&layout_60_ansi {
+    status = "disabled";
+};
+
+&layout_60_iso {
+    status = "disabled";
+};
+
+&layout_60_all1u {
+    status = "disabled";
+};
+
+&layout_60_hhkb {
+    status = "disabled";
+};
+
+/ {
+    60_percent_position_map {
```

Comment:
If someone wants to supplement these mappings in their board/shield where they add one more custom layout, they'll need a way to add new entries to the mapping, so we should give this labels folks can reference, e.g.:

```suggestion
    layouts_common_60percent_position_map: layouts_common_60percent_position_map {
```

Sorry for the late addition, just thought of this as I was going over this with fresh eyes.